src/store/actions/authApiStatusAction.js

```diff
@@ -0,0 +1,21 @@
+import * as types from "./actionTypes";
+import * as authApi from '../../utils/api/authApi';
+import setApiError from './setApiErrorAction';
+import { beginApiCall } from './apiStatusActions';
+
+export const authApiStatusSuccess = (payload) => ({ type: types.AUTH_API_STATUS__SUCCESS, payload });
+
+export function authApiStatus() {
+    return (dispatch) => {
+        dispatch(beginApiCall());
+        return authApi
+            .authApiStatus(() => dispatch(setApiError('authApiStatus', true)))
```

Comment:
This function doesn't take any arguments:
```suggestion
            .authApiStatus()
```